example/kotlinlib/web/2-todo-ktor/build.mill

```diff
@@ -0,0 +1,57 @@
+//// SNIPPET:BUILD
+package build
+import mill._, kotlinlib._
+
+object `package` extends RootModule with KotlinModule {
+
+  def kotlinVersion = "1.9.24"
+
+  def mainClass = Some("com.example.TodoMVCApplicationKt")
+
+  val ktorVersion = "2.3.12"
+  val exposedVersion = "0.53.0"
+
+  def ivyDeps = Agg(
+      ivy"io.ktor:ktor-server-core-jvm:$ktorVersion",
+      ivy"io.ktor:ktor-server-netty-jvm:$ktorVersion",
+      ivy"org.jetbrains.exposed:exposed-core:$exposedVersion",
+      ivy"org.jetbrains.exposed:exposed-jdbc:$exposedVersion",
+      ivy"com.h2database:h2:2.2.224",
+      ivy"io.ktor:ktor-server-webjars-jvm:$ktorVersion",
+      ivy"org.webjars:jquery:3.2.1",
+      ivy"io.ktor:ktor-server-thymeleaf-jvm:$ktorVersion",
+      ivy"org.webjars:webjars-locator:0.41",
+      ivy"org.webjars.npm:todomvc-common:1.0.5",
+      ivy"org.webjars.npm:todomvc-app-css:2.4.1",
+      ivy"ch.qos.logback:logback-classic:1.4.14"
+  )
+
+  object test extends KotlinModuleTests with TestModule.Junit5 {
+    def ivyDeps = super.ivyDeps() ++ Agg(
+      ivy"io.kotest:kotest-runner-junit5-jvm:5.9.1",
+      ivy"io.ktor:ktor-server-test-host-jvm:2.3.12"
+    )
+  }
+}
+
+// This example implementing the well known
+// https://todomvc.com/[TodoMVC] example app using Kotlin and Ktor. Apart from running a webserver,
+// this example also demonstrates:
+//
+// * Serving HTML templates using Thymeleaf
+// * Serving static Javascript and CSS using Webjars
+// * Querying a SQL database using Exposed
+// * Testing using a H2 in-memory database
+
+/** Usage
+
+ > mill test
+
+ > mill test.runBackground
+
+ > curl http://localhost:8080
+ ...<h1>todos</h1>...
+
```

Comment:
I think this can be fixed by de-denting all the lines in the usage block. The parser handling these is pretty fragile and doesn't tolerate extraneous whitespace